packages/forms/src/styled/tiles/StyledTileLabel.ts

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import styled from 'styled-components';
+import { retrieveComponentStyles, DEFAULT_THEME } from '@zendeskgarden/react-theming';
+
+const COMPONENT_ID = 'forms.tile_label';
+
+interface IStyledTileLabelProps {
+  isStacked?: boolean;
+}
+
+export const StyledTileLabel = styled.span.attrs({
+  'data-garden-id': COMPONENT_ID,
+  'data-garden-version': PACKAGE_VERSION
+})<IStyledTileLabelProps>`
+  display: block;
+  margin-top: ${props => (props.isStacked ? 0 : props.theme.space.base * 2)}px;
+  overflow: hidden;
+  text-align: ${props => !props.isStacked && 'center'};
+  text-overflow: ellipsis;
+  line-height: ${props => props.theme.space.base * 5}px;
```

Comment:
Should be using the `getLineHeight` utility here.

Reply:
Updated.